src/libraries/Microsoft.Extensions.Hosting/src/HostingHostBuilderExtensions.cs

```diff
@@ -134,6 +134,18 @@ public static IHostBuilder ConfigureContainer<TContainerBuilder>(this IHostBuild
             return hostBuilder.ConfigureContainer<TContainerBuilder>((context, builder) => configureDelegate(builder));
         }
 
+        /// <summary>
+        /// Adds a delegate for configuring the host settings. This can be called multiple times and
+        /// the results will be additive.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IHostBuilder"/> to configure.</param>
+        /// <param name="configureDelegate">The delegate for the <see cref="HostOptions"/>.</param>
+        /// <returns>The same instance of the <see cref="IHostBuilder"/> for chaining.</returns>
+        public static IHostBuilder ConfigureHostOptions(this IHostBuilder hostBuilder, Action<HostOptions> configureDelegate)
+        {
+            return hostBuilder.ConfigureServices((context, collection) => collection.Configure(configureDelegate));
```

Comment:
nit: you could be using the other overload here for simplicity, since no context is passed around

```suggestion
            return hostBuilder.ConfigureServices(collection => collection.Configure(configureDelegate));
```